### ISSUE TITLE
Allow user to kill instance

### ISSUE THREAD

**Issue by deliciouslytyped** [iblech/agdapad#2]
I keep breaking my instance in various ways, is there any way to force a restart of a given instance?
(Once by minimizing the emacs window, another time by somehow hanging emacs.)

This is very cool by the way, thanks. :)
I might finally learn emacs.

**Comment by iblech** [comment]
I agree that waiting for the idle timeout is not very nice when you want to play with Agda :-)

By switching to a second virtual desktop by Alt+2 and then pressing Alt+Shift+Enter, a terminal should pop up where you can kill the running emacs instance (for instance with `killall emacs` or `killall -9 emacs` in severe cases). This should then cause an immediate reset of the session.

I'll add it to my to do list to implement some better solution (do you have a suggestion for the user interface?) or at least document this one, but that might take a while.

I am happy that the Agdapad is useful to you :-)
